### PR TITLE
Bump devon_rex images from 2.11.0 to 2.12.0

### PR DESCRIPTION
**Pull request by yykamei**
Via:
```
$ bundle exec rake dockerfile:bump_devon_rex'[2.12.0]'
```

See also:
- https://github.com/sider/devon_rex/releases/tag/2.12.0
- https://github.com/sider/devon_rex/blob/2.12.0/CHANGELOG.md

### CODE REVIEW

File: test/smokes/rubocop/gem_install/install_from_sideci.yml_and_Gemfile.lock_when_gemspec_exists/test.gemspec [hunk 1]
```diff
@@ -19,6 +19,6 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 2.0.0"
+  spec.add_development_dependency "bundler", "~> 2.0"
```
Comment (yykamei) [inline test/smokes/rubocop/gem_install/install_from_sideci.yml_and_Gemfile.lock_when_gemspec_exists/test.gemspec]: We have to loosen the dependency here to make the test pass.

https://bundler.io/v1.2/gemfile.html

>  The specifier ~> has a special meaning, best shown by example. ~> 2.0.3 is identical to >= 2.0.3 and < 2.1. ~> 2.1 is identical to >= 2.1 and < 3.0.

**Comment by yykamei** [comment]
CI passed. I'll merge it after rebasing.

File: test/smokes/rubocop/gem_install/install_from_sideci.yml_and_Gemfile.lock_when_gemspec_exists/test.gemspec (hunk 1 above)
Comment (ybiquitous) [reply test/smokes/rubocop/gem_install/install_from_sideci.yml_and_Gemfile.lock_when_gemspec_exists/test.gemspec]: Does this dependency affect on actual analysis?

Comment (yykamei) [reply test/smokes/rubocop/gem_install/install_from_sideci.yml_and_Gemfile.lock_when_gemspec_exists/test.gemspec]: Maybe. We might have to install Bundler `2.1.x`, `2.0.x`, and `1.x` in case we can handle the various gemspec files.

Comment (ybiquitous) [reply test/smokes/rubocop/gem_install/install_from_sideci.yml_and_Gemfile.lock_when_gemspec_exists/test.gemspec]: Umm..., it may be a bug. I agree with the change to pass CI and release a new version, but could you please a new issue about this? We have to tackle it later.

Comment (yykamei) [reply test/smokes/rubocop/gem_install/install_from_sideci.yml_and_Gemfile.lock_when_gemspec_exists/test.gemspec]: I'm doing :running_man:

Comment (ybiquitous) [reply test/smokes/rubocop/gem_install/install_from_sideci.yml_and_Gemfile.lock_when_gemspec_exists/test.gemspec]: I want to believe that this is an edge case... 😅
